tests/integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -26,6 +26,9 @@ K8S_TEST_DEBUG="${K8S_TEST_DEBUG:-false}"
 K8S_TEST_HOST_TYPE="${K8S_TEST_HOST_TYPE:-small}"
 # Setting to "yes" enables fail fast, stopping execution at the first failed test.
 K8S_TEST_FAIL_FAST="${K8S_TEST_FAIL_FAST:-no}"
+# Setting greater than zero enables the retrying of failed tests. The value
+# of that variable is used for waiting between the executions.
+K8S_TEST_RETRY="${K8S_TEST_RETRY:-0}"
```

Comment:
How about `K8S_TEST_RETRY_TIMEOUT` to emphasize it represents a timeout? (the current name could be perceived as how many times to retry the test)

Reply:
I think `K8S_TEST_RETRY_AFTER_SEC` is best since there's only one retry.